### PR TITLE
User can now  view relevant race and track data via dropdown

### PR DESCRIPTION
**Pull request by aaronkeith**
Races 1-6 are currently linked and working.
f1ormula_tracks

### CODE REVIEW

**Comment by aaronkeith** [comment]
The API is lagging badly in the example .gif

File: public/main.js [hunk 1]
```diff
@@ -159,5 +134,86 @@ function renderDriverFinish({
...
+}
+
+function finish(roundNumber) {
+  fetch(
+    'http://ergast.com/api/f1/2017/' + roundNumber + '/results.json'
+  ).then(response =>
+    response
+      .json()
+      .then(driverResolve => {
+        //console.log(driverResolve)
```
Comment (thebearingedge) [inline public/main.js]: Watch out for stray comments!

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Just one comment and you're good!

**Comment by aaronkeith** [comment]
Shoot, I did go through and delete them as you asked, yet I did miss that one. I have deleted it.

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**